File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.js
function solution(my_string) {
    let answer = "";
    
    for (let i = 0; i < my_string.length; i++) {
        let charCode = my_string.charCodeAt(i);
        
        // 대문자인 경우 (ASCII 코드가 65~90)
        if (charCode < 97) {
            answer += String.fromCharCode(charCode + 32);  // 소문자로 변환
        } else {
            answer += String.fromCharCode(charCode - 32);  // 대문자로 변환
        }
    }

    return answer;
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.py
def solution(my_string):
    answer = ""
    
    for char in my_string:
        char_code = ord(char)
        
        # 대문자인 경우 (ASCII 코드가 65~90)
        if char_code < 97:
            answer += chr(char_code + 32)  # 소문자로 변환
        else:
            answer += chr(char_code - 32)  # 대문자로 변환
    
    return answer